lightning/src/util/logger.rs

```diff
@@ -17,11 +17,13 @@
 use core::cmp;
 use core::fmt;
 
-static LOG_LEVEL_NAMES: [&'static str; 5] = ["TRACE", "DEBUG", "INFO", "WARN", "ERROR"];
+static LOG_LEVEL_NAMES: [&'static str; 6] = ["GOSSIP", "TRACE", "DEBUG", "INFO", "WARN", "ERROR"];
 
 /// An enum representing the available verbosity levels of the logger.
 #[derive(Copy, Clone, PartialEq, Eq, Debug, Hash)]
 pub enum Level {
+	/// Designates extremely verbose information, including gossip-induced messages
+	Gossip,
 	/// Designates very low priority, often extremely verbose, information
 	Trace,
 	/// Designates lower priority information
```

Comment:
We may want to consider category-based logging instead of level-based logging in the future. Or maybe both? No need to change here.

Reply:
Hum, but if I'm not mistaken a `Gossip` log level seems to be required for now to log gossip events separately?

Reply:
Yup, no need to change anything in this PR. Just pointing out that we may want to rethink this in the future.

Reply:
Great, thanks!